contracts/tokens/ERC1155BridgeToken.sol

```diff
@@ -78,7 +80,21 @@ contract ERC1155BridgeToken is ERC1155, IBurnableMintableERC1155Token {
         uint256[] memory _tokenIds,
         uint256[] memory _values
     ) external override onlyBridge {
-        _mintBatch(_to, _tokenIds, _values, new bytes(0));
+        if (_tokenIds.length == 1 && _values.length == 1) {
+            _mint(_to, _tokenIds[0], _values[0], new bytes(0));
+        } else {
+            if (!hasAlreadyMinted) {
```

Comment:
May we add a short description why it is needed as so it will be clear for auditor in the future why we decided to complicate the operation.